### ISSUE TITLE
Could we have feature to subscrive  to any RSS feeds?

### ISSUE THREAD

**Issue by SupernovaTitanium** [TideDra/zotero-arxiv-daily#21]
Hi, we can now follow the sub-category of arXiv. Can we follow any RSS feeds since some new papers are not only from arXivs.

**Comment by TideDra** [comment]
Good idea! But I'm not familiar with RSS and not sure whether it is possible to add any RSS source to the workflow. Information from different sources should be able to be organized in a unified structured form, so that we can easily extract the paper's metadata. Or if you're an RSS expert, welcome the contribution :)
